generators/app/templates/template/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "watch": "onchange '**/*.d.ts' -e 'out/**' -- npm run build",
     "build": "typings bundle<%- ambient %> -o out",
-    "test": "ts-node test/test.ts"
+    "test": "ts-node test/test.ts",
+    "postinstall": "typings install"
```

Comment:
Yeah, `postinstall` runs any time someone installs the package (which means anyone installing `generator-typings` from NPM will fail if they don't have Typings installed, don't worry - you aren't the first person to overlook this :wink:). If you use `prepublish`, it occurs after `npm install` (legacy reasons) and before publishing to NPM so it's usually ideal.